src/main/java/appeng/services/VersionChecker.java

```diff
@@ -76,16 +76,24 @@ public void run()
 	{
 		Thread.yield();
 
-		final String rawLastCheck = this.config.lastCheck();
-		final long lastCheck = Long.parseLong( rawLastCheck );
-		final Date now = new Date();
-		final long nowInMs = now.getTime();
-		final long intervalInMs = this.config.interval() * SEC_TO_HOUR * MS_TO_SEC;
-		final long lastAfterInterval = lastCheck + intervalInMs;
+		try
+		{
+			final String rawLastCheck = this.config.lastCheck();
+			final long lastCheck = Long.parseLong( rawLastCheck );
+			final Date now = new Date();
+			final long nowInMs = now.getTime();
+			final long intervalInMs = this.config.interval() * SEC_TO_HOUR * MS_TO_SEC;
+			final long lastAfterInterval = lastCheck + intervalInMs;
+
+			this.processInterval( nowInMs, lastAfterInterval );
 
-		this.processInterval( nowInMs, lastAfterInterval );
+			AELog.info( "Stopping AE2 VersionChecker" );
+		}
+		catch( Exception e )
```

Comment:
This will never reach the main thread. unchecked exceptions are bugs, why catching them unless you do not want them being reported. But I guess here is fine since its `AELog.error()`